model/indices/stats/operations.smithy

```diff
@@ -8,9 +8,8 @@ $version: "2"
 namespace OpenSearch
 use opensearch.openapi#vendorExtensions
 
-// TODO: Fill in API Reference URL
 @externalDocumentation(
-    "API Reference": "https://opensearch.org/docs/latest"
+    "API Reference": "https://opensearch.org/docs/latest/im-plugin/data-streams/"
```

Comment:
This is not the correct URL for this API, I do not believe there's any info on the website docs about this one.